pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -5883,8 +5883,8 @@ func TestChangefeedErrors(t *testing.T) {
 		`webhook-https://fake-host`,
 	)
 	sqlDB.ExpectErrWithTimeout(
-		t, `this sink is incompatible with option compression`,
```

Comment:
can you add another test case that returns the "this sink is incompatible with option compression" error, with another sink eg pubsub.

Reply:
done